sdv/constraints/tabular.py

```diff
@@ -1020,6 +1020,7 @@ def _sample_constraint_columns(self, table_data):
         has_one = table_data[condition_columns].sum(axis=1) == 1.0
```

Comment:
Not really part of this PR, but we could think about doing the fix here:
It seems like we are accessing `table_data[condition_columns]` several times, and also computing `sum(axis=1)` on them at least twice. Could we capture these in variables?